JSDemos/Demos/Scheduler/Timelines/description.md

```diff
@@ -1 +1,10 @@
-This demo illustrates the collection of&nbsp;timeline views available in&nbsp;the Scheduler component. Appointments are associated with several resources (an&nbsp;appointment color depends on&nbsp;the associated resource) and grouped by&nbsp;one of&nbsp;them.
\ No newline at end of file
+Timeline views display appointments as a sequence of events on a horizontal timeline, as opposed to basic views that display appointments in a calendar-like fashion. You can use the View Switcher above the timetable to switch between the views.
```

Comment:
Since some readers may not be familiar with the phrase "in a calendar-like fashion", can we change it to "in a calendar format"?